test/functional/build.gradle.kts

```diff
@@ -1,5 +1,60 @@
+import com.moowork.gradle.node.yarn.YarnTask
+import java.util.function.Consumer
+
 plugins {
     id("java")
+    id("com.cognifide.aem.instance")
+    id("com.moowork.node")
 }
 
 description = "Example - Functional Tests"
+
+tasks{
+    register<YarnTask>("testFunctional") {
+        dependsOn("testSetup", "testInstallNode")
+        group = "check"
+        setYarnCommand("jest")
+        setWorkingDir(file("${projectDir.absolutePath}/test.functional"))
+        outputs.dir(file("${projectDir.absolutePath}/test.functional"))
+        doFirst {
+            val props = mutableListOf()
+            if (project.hasProperty("aem.env")) {
+                props.addAll(listOf("--config", "./env/${project.properties["aem.env"]}.config.js"))
+            } else {
+                props.addAll(listOf("--config", "./env/local-publish.config.js"))
+            }
+            args = props
+        }
+    }
+
+    register("testSetup") {
```

Comment:
this task could be removed and be incorporated into `runJestPuppeteer` / splitting could be a source of errors if someone decide to configure input/outputs of this task so that caching could cause this props not regenerated.